pkg/arvo/ted/migrate-group.hoon

```diff
@@ -0,0 +1,79 @@
+/-  spider, *metadata-store, *publish
+/+  strandio
+=,  strand=strand:spider
+^-  thread:spider
+|=  arg=vase
+=/  m  (strand ,vase)
+^-  form:m
+=/  [og-path=path ng-path=path ~]  !<([path path ~] arg)
+;<  bol=bowl:spider  bind:m  get-bowl:strandio
+::
+=/  gs-path  /(scot %p our.bol)/group-store/(scot %da now.bol)
+=/  ms-path  /(scot %p our.bol)/metadata-store/(scot %da now.bol)
+=/  p-path   /(scot %p our.bol)/publish/(scot %da now.bol)
+::
+=/  og=(unit (set ship))
+  .^((unit (set ship)) %gx :(weld gs-path og-path /noun))
+?~  og
+  (pure:m !>("no such group: {<og-path>}"))
+=/  ng=(unit (set ship))
+  .^((unit (set ship)) %gx :(weld gs-path ng-path /noun))
+?~  ng
+  (pure:m !>("no such group: {<ng-path>}"))
+::
+=/  ass=associations  .^(associations %gx :(weld ms-path /group og-path /noun))
+=/  ass-list=(list [[group-path resource] metadata])  ~(tap by ass)
```

Comment:
```suggestion
=/  assoc-list=(list [[g-path=group-path res=resource] meta=metadata])
  %~  tap  by
  .^(associations %gx :(weld ms-path /group og-path /noun))
```

If you add faces to the list type here, you can just `=,` them in below, instead of `=/`ing and writing out the type twice. Feels better imo, but also nbd.